lib/tasks/release-notes.js

```diff
@@ -0,0 +1,27 @@
+module.exports.displayReleaseNotes = async (ctx, task) => {
+    const got = require('got');
+    const response = await got('https://api.github.com/repos/TryGhost/Ghost/releases', {json: true});
+    const relevantNotes = response.body.filter(note => note.tag_name === ctx.version)[0];
+
+    task.title = 'Fetched release notes';
+
+    if (!relevantNotes) {
+        return;
+    }
+
+    ctx.releaseNotesContainWarning = relevantNotes.body.includes('⚠️');
+    task.output = '\n' + relevantNotes.body + '\n';
+};
+
+module.exports.checkForConfirmation = async ({ui}) => {
+    const {CliError} = require('../errors');
+
+    const question = 'The release notes contain a ⚠️, which indicates an issue with the release. Are you sure you want to continue?';
+    const answer = await ui.confirm(question, true);
+
+    if (answer) {
+        return Promise.resolve();
+    }
+
+    throw new CliError('');
```

Comment:
WIP

Reply:
probably this, just give it an informative error message